apps/omg_watcher/lib/eventer/event.ex

```diff
@@ -93,13 +93,34 @@ defmodule OMG.Watcher.Eventer.Event do
 
     def name, do: "invalid_exit"
 
-    defstruct [:amount, :currency, :owner, :utxo_pos]
+    defstruct [:amount, :currency, :owner, :utxo_pos, :eth_height]
 
     @type t :: %__MODULE__{
             amount: pos_integer(),
             currency: binary(),
             owner: binary(),
-            utxo_pos: pos_integer()
+            utxo_pos: pos_integer(),
+            eth_height: pos_integer()
+          }
+  end
+
+  defmodule UnchallengedExit do
+    @moduledoc """
+    Notifies about an invalid exit, that is dangerously approaching finalization, without being challneged
+
+    It is a prompt to exit
+    """
```

Comment:
If you added `It is a prompt to exit` to `UnchallengedExit` maybe it is worth to add this to other `byzantine_bad_chain` events(invalid_block etc) to be consistent with what is in README.md

Reply:
it is there, line 465, to cover the whole section